mkosi/resources/mkosi.md

```diff
@@ -2754,6 +2730,33 @@ Note that the minimum required Python version is 3.9.
   `systemd-homed-firstboot.service` will prompt to create a regular user
   on first boot if there are no regular users.
 
+- Why do I see failures to chown files when building images?
+
+  When not running as root, your user is not able to change ownership of
+  files to arbitrary owners. Various distributions still ship files in their
+  packages that are not owned by the root user. When not running as root, mkosi
+  maps the current user to root when invoking package managers, which means that
+  changing ownership to root will work but changing ownership to any other user
+  or group will fail.
+
+  If this behavior causes applications running in your image to misbehave, you
+  can consider running `mkosi` as root which avoids this problem. Alternatively,
+  if running `mkosi` as root is not desired, you can use
+  `unshare --map-auto --map-current-user --setuid 0 --setgid 0` to become root in
+  a user namespace with more than one user assuming the UID/GID mappings in
+  `/etc/subuid` and `/etc/subgid` are configured correctly. Note that running mkosi
+  as root or with `unshare` means that all output files produced by mkosi will not
+  be owned by your current user anymore.
+
+  Note that for systemd services that need directories in `/var` owned by the service
+  user and group, an alternative to shipping these directories in rpm packages or
+  creating them via systemd-tmpfiles is to use `StateDirectory=`, `CacheDirectory=` or
```

Comment:
Creating them via tmpfiles.d config wouldn't be bad as long as the config is shipped in `/usr/` as well,  the problem are postinst scriptlets, so I'd write
```suggestion
  creating them via package postinstallation scripts, is to use `StateDirectory=`, `CacheDirectory=` or
```